src/tetris/scene/mainScene.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference path="../../../definitions/phaser.d.ts"/>
+
+import Field from "tetris/field/field";
+import Player from "tetris/player/player";
+import BrickFactory from "tetris/brick/brickFactory";
+import BiasEngine from "tetris/BiasEngine/BiasEngine";
+
+const GAME_WIDTH: number = 18;
```

Comment:
Maybe rename to something like `FIELD_WIDTH` and `FIELD_HEIGHT` to prevent confusion with the actual game-canvas size?

Reply:
I agree - was my bad, sorry.